content/usgs-packages/dataRetrieval_Discovery.Rmd

```diff
@@ -66,116 +66,84 @@ For more query parameters, visit [NWIS service documentation](https://waterservi
-  state_sites <- whatNWISsites(stateCd=i, siteType="ST", parameterCd="00010")
-  sites_sw_stream_temp <- bind_rows(sites_sw_stream_temp, state_sites)
+  state_data <- whatNWISdata(stateCd=i, siteType="ST", parameterCd="00010")
+  data_sw_stream_temp <- bind_rows(data_sw_stream_temp, state_data)
 }
 
-nrow(sites_sw_stream_temp)
+nrow(data_sw_stream_temp)
 ```
 
-### whatNWISdata
-
-`whatNWISdata` will return a data.frame specifying the types of data available for a specified site(s) that fits your querying criteria. You can add queries by the data service, USGS parameter code, or statistics code. Continuing with the South Carolina temperature data example from the `whatNWISsites` section,  let's look for the mean daily stream temperature. Since you cannot query `whatNWISdata` using `siteType` or `stateCd`, let's use the site numbers from the previous lesson to specify South Carolina stream sites. You will need to specify the parameter again since the sites likely collect more than one parameter.
+Continuing with the South Carolina temperature data example, let's look for the mean daily stream temperature.  
 
 ```{r data_sc_stream_temp_avg, message=FALSE, warning=FALSE}
 # Average daily SC stream temperature data
 data_sc_stream_temp_avg <- whatNWISdata(
-  siteNumbers = unique(sites_sc_stream_temp[['site_no']]),
+  stateCd="SC", 
+  siteType="ST",
   parameterCd="00010",
   service="dv",
   statCd="00003")
 nrow(data_sc_stream_temp_avg)
 ```
 
-In some queries, you might notice an error that says "Too Long". This generally happens when you specify too many site numbers at once, which makes the web service URL too long. If this happens, try shortening the number of sites you are querying at a time and append all results at the end (similar to how we queried each state separately in the `whatNWISsites` example). You could also use a different major filter (bounding box, state, huc) that would give a shorter list to include in your query URL.
-
-Let's look at the dataframe returned from `whatNWISdata`:
-
-```{r data_sc_stream_temp_avg_explore, message=FALSE, warning=FALSE}
-head(data_sc_stream_temp_avg)
-```
-
-The data returned from this query can give you information about the data available for each site including, date of first and last record (`begin_date`, `end_date`), number of records (`count_nu`), site altitude (`alt_va`), corresponding hydrologic unit code (`huc_cd`), and parameter units (`parameter_units`). These columns allow even more specification of data requirements before actually downloading the data. 
+In some queries, you might notice an error that says "Too Long". This generally happens when you specify too many site numbers at once, which makes the web service URL too long. If this happens, try shortening the number of sites you are querying at a time and append all results at the end. You could also use a different major filter (bounding box, state, huc) that would give a shorter list to include in your query URL.
```

Comment:
Have you noticed this lately? We switched from `GET` to `POST` which should have eliminated the "too long" problem

Reply:
I have not noticed it lately, but see some of the effects here https://github.com/USGS-R/dataRetrieval/pull/362/files. Cool, I will remove this statement!